charts/sourcegraph/tests/storageClass_test.yaml

```diff
@@ -12,3 +12,11 @@ tests:
   - equal:
       path: parameters.zones
       value: us-central1-f
+- it: should not have 'type' when storageClass.type=null
+  set:
+    storageClass:
+      create: true
+      type: null
+  asserts:
+  - isNull:
+      path: parameters
```

Comment:
helm unittest has no easy way to check if a map does not contain a key or not

is there a better way to test?